packages/horizon/README.md

```diff
@@ -0,0 +1,13 @@
+# Sample Hardhat Project
+
+This project demonstrates a basic Hardhat use case. It comes with a sample contract, a test for that contract, and a script that deploys that contract.
```

Comment:
We're far from a basic hardhat use case now 🤣 
(let's make a note to update this README in a future PR)